app/views/application_letters/index.html.erb

```diff
@@ -1,6 +1,6 @@
 <%- model_class = ApplicationLetter -%>
 <div class="page-header">
-  <h1><%=t '.title', :default => model_class.model_name.human.pluralize.titleize %></h1>
+  <h1><%=t '.title', :default => model_class.model_name.human(:count => model_class.count).titleize %></h1>
```

Comment:
I think right now it takes the number of all application letters in order to decide between singular and plural. Is this intentioned?